### ISSUE TITLE
Add recipes for Bronze, Brass, and other alloys to the Alloy Mixer

### ISSUE THREAD

**Issue by drsaeva** [Aragas/angelsextended-remelting#3]
It would be cool to add recipes for Bronze, Brass, and such to the alloy mixer to enable folks to chose whether to pipe or belt around the component metals. If you're cool with it, I'd be happy to submit a PR for the change myself.

**Comment by Aragas** [comment]
Sure!
Were there new recipe additions? I was under impression that I covered everything at that point!

**Comment by Medved2k** [comment]
Yep very missed to recipes for alloys, bronze, brass and other alloys.

**Comment by Medved2k** [comment]
I've done with some experiments in fork branch - and found that alloys existed recipes aren't been enabled even with all technologies opened. So I've added them into tech research tree manually and they have appeared. 
And there are some modifications to steel production - there are 4 more recipes to add.

**Comment by drsaeva** [comment]
Late replying, but awesome! Thanks for checking it out @Medved2k . I took a little bit of time looking into it myself, but I'm still learning how to grok Factorio modding so it ended up getting sidelined.  I remember it looked like some of the recipes were there, but I think there were tech mismatches for some things. Sounds like it should be square, you should put up that PR!

**Comment by Medved2k** [comment]
> Late replying, but awesome! Thanks for checking it out @Medved2k . I took a little bit of time looking into it myself, but I'm still learning how to grok Factorio modding so it ended up getting sidelined. I remember it looked like some of the recipes were there, but I think there were tech mismatches for some things. Sounds like it should be square, you should put up that PR!

Sorry for been late, but I've created PR.

**Comment by Aragas** [comment]
Should be fixed
